setup.py

```diff
@@ -26,7 +26,6 @@ def read_version(package):
     'requests>=2.27,<3',
     'inflection>=0.3.1,<0.6',
     'werkzeug>=2,<3',
-    'importlib-metadata>=1 ; python_version<"3.8"',
     'packaging>=20',
```

Comment:
I think we can drop `packaging` as a dependency as well.

Reply:
I'll remove it 👍